pkg/asset/machines/gcp/machines.go

```diff
@@ -113,7 +113,7 @@ func provider(clusterID string, platform *gcp.Platform, mpool *gcp.MachinePool,
 			Email:  fmt.Sprintf("%s-%s@%s.iam.gserviceaccount.com", clusterID, role[0:1], platform.ProjectID),
 			Scopes: []string{"https://www.googleapis.com/auth/cloud-platform"},
 		}},
-		Tags:        []string{fmt.Sprintf("%s-%s", clusterID, role)},
+		Tags:        append(mpool.Tags, []string{fmt.Sprintf("%s-%s", clusterID, role)}...),
```

Comment:
nit: I think this line is doing too much. Would be better, imho, to move the consolidation of tags out of the struct literal. 

but I don't think this is necessary to change unless you want to or are making other changes.